src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -183,17 +205,58 @@ function TextArea({
   containerRef,
   ...restProps
 }: TextAreaProps & JSX.TextareaHTMLAttributes<HTMLTextAreaElement>) {
+  const [popoverOpen, setPopoverOpen] = useState(false);
+  const textareaRef = useSyncedRef(containerRef);
+  const store = useSidebarStore();
+  const atMentionsEnabled = store.isFeatureEnabled('at_mentions');
```

Comment:
I expect we will want to reuse editor components outside the sidebar in future. This will be easier if there is no sidebar dependency to untangle, by passing in `atMentionsEnabled` as a prop.